### PR TITLE
Extract multiple docker tags

### LINKED ISSUE #130: Extract multiple Docker tags
For FerretDB/FerretDB#70, we want `extract-docker-tag` action to be able to return several tags. For PR branches, we should return a single tag like now, but for pushed git tags, we should return two Docker tags: version without `v` prefix and `latest`. To simplify migration, that probably should be a separate output field. It will be used by that code in FerretDB repo:
* https://github.com/FerretDB/FerretDB/blob/d48b4fa526675440c534d56bea291a0a652bd4c5/.github/workflows/docker.yml#L97-L98
* https://github.com/FerretDB/FerretDB/blob/d48b4fa526675440c534d56bea291a0a652bd4c5/Taskfile.yml#L421-L449

Add a unit test or test it with your own fork.

### PR DESCRIPTION
**Pull request by w84thesun**
Closes #130.

### CODE REVIEW

File: extract-docker-tag/main.go [hunk 1]
```diff
@@ -44,13 +44,15 @@ func main() {
...
 }
 
 type result struct {
-	owner string // ferretdb
-	name  string // github-actions-dev
-	tag   string // pr-add-features or 0.0.1
-	ghcr  string // ghcr.io/ferretdb/github-actions-dev:pr-add-features or ghcr.io/ferretdb/github-actions-dev:0.0.1
+	owner      string // ferretdb
+	name       string // github-actions-dev
+	tag        string // pr-add-features or 0.0.1
```
Comment (AlekSi) [inline extract-docker-tag/main.go:53]:
```suggestion
	tags        []string // {"pr-add-features", "0.0.1"}
```

File: extract-docker-tag/main.go [hunk 2]
```diff
@@ -44,13 +44,15 @@ func main() {
...
+	ghcr       string // ghcr.io/ferretdb/github-actions-dev:pr-add-features or ghcr.io/ferretdb/github-actions-dev:0.0.1
```
Comment (AlekSi) [inline extract-docker-tag/main.go:54]: Keep as-is to make migration easier

File: extract-docker-tag/main.go [hunk 3]
```diff
@@ -44,13 +44,15 @@ func main() {
...
+	ghcrLatest string // ghcr.io/ferretdb/github-actions-dev:latest (only for pushed tags)
```
Comment (AlekSi) [inline extract-docker-tag/main.go:55]:
```suggestion
	ghcrImages []string // {"ghcr.io/ferretdb/github-actions-dev:pr-add-features"} or {"ghcr.io/ferretdb/github-actions-dev:0.0.1", "ghcr.io/ferretdb/github-actions-dev:latest"}
```

File: extract-docker-tag/main.go [hunk 4]
```diff
@@ -42,19 +42,22 @@ func main() {
 
 	action.SetOutput("owner", result.owner)
 	action.SetOutput("name", result.name)
-	action.SetOutput("tag", result.tag)
+	action.SetOutput("tag", strings.Join(result.tags, ","))
```
Comment (AlekSi) [inline extract-docker-tag/main.go:45]:
```suggestion
	action.SetOutput("tags", strings.Join(result.tags, ","))
```

We also should update action.yml file

File: extract-docker-tag/main.go [hunk 5]
```diff
@@ -78,7 +81,7 @@ func extract(action *githubactions.Action) (*result, error) {
 		// always add tag prefix and name suffix to prevent clashes on "main", "latest", etc
 		branch := action.Getenv("GITHUB_HEAD_REF")
 		parts = strings.Split(strings.ToLower(branch), "/") // for branches like "dependabot/submodules/XXX"
-		result.tag = "pr-" + parts[len(parts)-1]
+		result.tags = append(result.tags, "pr-"+parts[len(parts)-1])
```
Comment (AlekSi) [inline extract-docker-tag/main.go:84]: I think simple assignment will be better there

File: extract-docker-tag/main.go [hunk 6]
```diff
@@ -117,10 +124,12 @@ func extract(action *githubactions.Action) (*result, error) {
 		return nil, fmt.Errorf("unhandled event type %q", event)
 	}
 
-	if result.tag == "" {
+	if len(result.tags) == 0 {
 		return nil, fmt.Errorf("failed to extract tag for event %q", event)
```
Comment (AlekSi) [inline extract-docker-tag/main.go:128]:
```suggestion
		return nil, fmt.Errorf("failed to extract tags for event %q", event)
```

File: extract-docker-tag/main.go (hunk 5 above)
Comment (w84thesun) [reply extract-docker-tag/main.go]: Made the first assignment simple through that function.

**Review by AlekSi — COMMENTED** [review]
https://github.com/FerretDB/github-actions/pull/133#discussion_r1105919608

File: extract-docker-tag/main.go (hunk 4 above)
Comment (w84thesun) [reply extract-docker-tag/main.go]: Missed it, fixed it now.

**Review by AlekSi — APPROVED** [review]

**Pull request merged**